### ISSUE TITLE
Station Music Muted but still playing songs?

### ISSUE THREAD

**Issue by JanieUwU** [AzuraCast/AzuraCast#2690]
**Using Docker installation method**
Yes

**Host Operating System**
Debian 10

**Describe the bug**
Audio from public page/stream link is muted but station is still playing

**To Reproduce**
Just let the station play as normal, nothing was changed.

**Expected behavior**

**Relevant Logs**

No logs found that seemed abnormal
**Screenshots**

**Device(s):**
 - Device:  
 - OS: 
 - Browser: 
 - Version: 

**Additional context**
Restarting station seemed to fix it

**Comment by SlvrEagle23** [comment]
@JanieUwU What browser and client-side OS are you using, and have you had a chance to test this out with other browser/OS combinations to see if the issue persists?

I can't seem to reproduce this on the latest Chrome on Windows 10...clicking "mute" or dragging the volume slider all the way to zero properly mutes the audio track, though it of course continues playing silently in the background (only pausing avoids this).

**Comment by JanieUwU** [comment]
@SlvrEagle23 I used both latest Chrome versions on Windows10 and Android. I also have a discord bot that connects to the audio stream to play the station there all 3 were silent. The audio slider didn't change it, all the way/down no music was heard. I have no idea what caused it, no warning or errors in the logs. It was playing normally like I had it since I installed last week and first time it done this. Restarting fixed it so far though I am not sure how often it would happen again. I wish I could give more info on how to reproduce but I just woke up and saw it wasn't having audio come out, it changed songs and everything normally but still was silent.
